keps/3589-manage-jobs-selectively/README.md

```diff
@@ -110,6 +117,16 @@ managedJobsNamespaceSelector:
   matchLabels:
     kueue.x-k8s.io/managed-namespace: true
 ```
+#### Story 2
+
+Cluster admins want to opt in only selected namespaces to Kueue management. A user in a namespace not selected by managedJobsNamespaceSelector should not be able to submit a Kueue-managed workload by manually setting the queue-name label. This change ensures that only workloads in opted-in namespaces are ever reconciled, regardless of how they are labeled.
```

Comment:
My initial thought for the story was to just prevent creating the Jobs in the namespaces excluded by `managedJobsNamespaceSelector` with VAP if the Jobs have `queue-name` label. However, adding VAP for multiple Job types can be tedious and error prone. Also, the VAP-based approach is still ok even with this KEP.